internal/api/api_test.go

```diff
@@ -358,6 +358,31 @@ func TestGitHubTeamApproverCleansUpOldIgnoredReviewsComments(t *testing.T) {
 		ExpectedReviewRequestsMadeForFoo()
 }
 
+func TestGitHubTeamApproverCleansUpOldInvalidReviewsComments(t *testing.T) {
```

Comment:
Hmm, this test and the above tests makes me wonder: what happens when GitHub Team Approver encounters ignored & invalid reviewers in the same PR?

I think it will correctly clean-up comments according to the title, but if it's easy, might be nice to add a test that covers that use-case.

Reply:
Yes, it is based on comment title